src/components/actionSheet/actionSheet.js

```diff
@@ -42,8 +42,8 @@ function getPosition(options, dlg) {
 
     pos.left += (pos.width || 0) / 2;
 
-    const height = dlg.offsetHeight || 300;
-    const width = dlg.offsetWidth || 160;
+    const height = dlg.nominalOffsetHeight || 300;
```

Comment:
We may want to include fallbacks here for the normal `offsetHeight` and `offsetWidth`.

```js
dlg.nominalOffsetHeight || dlg.offsetHeight || 300;
```